[DOCS] Adds default inference endpoints information (#118463)
elastic/elasticsearch: docs/reference/inference/inference-apis.asciidoc

For more information about adaptive allocations and resources, refer to the {ml-docs}/ml-nlp-auto-scale.html[trained model autoscaling] documentation.

[discrete]
[[default-enpoints]]
=== Default {infer} endpoints

> Your {es} deployment contains some preconfigured {infer} endpoints that makes it easier for you to use them when defining `semantic_text` fields or {infer} processors.

Reviewer: ```suggestion
Your {es} deployment contains preconfigured {infer} endpoints which makes them easier to use when defining `semantic_text` fields or using {infer} processors.
```

Reviewer: minor rewording for clarity
